src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let count = 0;
+
+  for (let i = 0; i < days; i++) {
+    count += 40;
+  }
```

Comment:
```suggestion
  let count = days * 40;
```
Для чого цикл тут використовувати

Reply:
Agree, you don't need cycle here